packages/sampler/stories/qa/ExpandableInput.js

```diff
@@ -9,15 +9,16 @@ import {action} from '../../src/utils';
 ExpandableInput.displayName = 'ExpandableInput';
 
 const iconNames = ['', ...Object.keys(icons)];
+const sortedIconNames = ['', ...iconNames.sort()];
```

Comment:
This doubles the empty string.  Can simplify this change to just:
```
const iconNames = ['', ...Object.keys(icons).sort()];
```

And then you won't have to change the name used below.

Reply:
Alternatively: `iconNames.sort();`

Reply:
fixed with:
```const sortedIconNames = ['', ...iconNames.sort()];```
else the blank selection does not show with:
```const sortedIconNames = iconNames.sort();```

Reply:
Is the blank just sorted to the end of the list? So now there are 2 blanks, one at the top and one at the bottom?

Reply:
Blank is at the top only in the knob in the dropdown

Reply:
It must be getting pruned out elsewhere in the code. Because line 11 adds a blank (empty string) and then this line, 12, adds another blank (empty string).